### ISSUE TITLE
Tuple with TypeVar as Function Parameter: Expected class type but received "Literal[0]"

### ISSUE THREAD

**Issue by FunkMonkey** [microsoft/pylance-release#2169]
## Environment data

-   Language Server version: 2021.12.1
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): 3.96 64 bit Conda

## Expected behaviour

No error when using a parameter of a tuple type that uses a TypeVar

## Actual behaviour

Getting error when using a parameter of a tuple type that uses a TypeVar, see code below.

It seems to be a regression. I rolled back to Pylance 2021.11.2, which does not give me this error.

## Code Snippet / Additional information

```python
TNodeData = TypeVar("TNodeData")
TestTuple = Tuple[int, TNodeData]

test1: TestTuple[str] = (34234, "hello")
x = test1[0] # no error as expected

def some_func(node: TestTuple[str]):
    some_var= node[0] # unexpected error: Expected class type but received "Literal[0]"
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by bschnurr** [comment]
This issue has been fixed in version 2021.12.2, which we've just released. You can find the changelog here:  [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2021122-13-december-2021)
